TA_Scheduler_App/courseFeatures.py

```diff
@@ -50,30 +61,33 @@ def delete_section(sectionKey:int):
     #         return False
 
     @staticmethod
-    def edit_course(courseKey:int, newCourseName:str=""):
+    def edit_course(courseKey:int, newCourseName:str="",newSemester:str = ""):
         try:
             course = Course.objects.get(pk=courseKey)
         except Course.DoesNotExist:
             return False
-
-
-        if newCourseName is not '' and newCourseName is not None:
-            course.courseName = newCourseName
+        if newCourseName and newCourseName.strip():
+            course.courseName = newCourseName.strip()
+            course.save()
+        if newSemester and newSemester.strip():
+            course.semester = newSemester.strip()
             course.save()
 
         return course
 
     @staticmethod
-    def edit_section(sectionKey:int, newSectionTime:str=None):
+    def edit_section(sectionKey:int, newSectionCode: str,newInstructor: str):
         try:
             section = Section.objects.get(pk=sectionKey)
         except Section.DoesNotExist:
             return False
 
-        if newSectionTime is not None and newSectionTime is not '':
-            section.sectionTime = newSectionTime
+        if newSectionCode and newSectionCode.strip():
```

Comment:
```suggestion
        if  newSectionCode.strip():
```
ONLY IF we change the arguments to have `""` as defaults. We might also want to add conditionals if we want `sectionCode` to be a required field...